views/groups.handlebars

```diff
@@ -1,20 +1,24 @@
-<body>
-  <nav class="navbar navbar-default">
-    <div class="container-fluid">
-      <div class="navbar-header">
+<body><div id="groups-whole-page">
```

Comment:
see the div with id="groups-whole-page"? It is for when I basically want to do some CSS stuff with <body></body>, but I don't know if there are some special functionality with <body> from handlebars, so I figure this is safer.